docs/mimxrt/general.rst

```diff
@@ -0,0 +1,94 @@
+.. _mimxrt_general:
+
+General information about the MIMXRT port
+=========================================
+
+The i.MXRT MCU family is a high performance family of devices made by
+NXP. Based on a ARM7 core the provide may on-chip I/O units for building
+small to medium sized devices.
+
+Multitude of boards
+-------------------
+
+There is a multitude of modules and boards from different sources which carry
+a i.MXRT chip. MicroPython tries to provide a generic port which would run on
+as many boards/modules as possible, but there may be limitations. The
+NXP IMXRT1020-EVK and the Teensy 4.0 and Teensy 4.1 development boards are taken
+as reference for the port (for example, testing is performed on them).
+For any board you are using please make sure you have a data sheet, schematics
+and other reference materials so you can look up any board-specific functions.
+ 
+The following boards are supported by the port out of the box:
+
+- MIMXRT1010-EVK
+- MIMXRT1020-EVK
+- MIMXRT1050-EVK
+- MIMXRT1060-EVK
+- MIMXRT1064-EVK
+- Teensy 4.0
+- Teensy 4.1
+
+Supported MCUs
+--------------
+
++-------------+--------------------+-------------------------+
+| Product     | CPU                | Memory                  |
++=============+====================+=========================+
+| i.MX RT1064 | Cortex-M7 @600 MHz | 1 MB SRAM, 4 MB Flash   |
++-------------+--------------------+-------------------------+
+| i.MX RT1061 | Cortex-M7 @600 MHz | 1 MB SRAM               |
++-------------+--------------------+-------------------------+
+| i.MX RT1062 | Cortex-M7 @600 MHz | 1 MB SRAM               |
++-------------+--------------------+-------------------------+
+| i.MX RT1050 | Cortex-M7 @600 MHz | 512 kB SRAM             |
++-------------+--------------------+-------------------------+
+| i.MX RT1020 | Cortex-M7 @500 MHz | 256 kB SRAM             |
++-------------+--------------------+-------------------------+
+| i.MX RT1010 | Cortex-M7 @500 MHz | 128 kB SRAM             |
++-------------+--------------------+-------------------------+
+
+Note
+    Most of the controllers do not have internal flash memory. Therefore
+    their flash capacity is dependent on an external flash chip!
```

Comment:
In general I suggest that exclamation marks are not used in documentation unless really warranted.

Reply:
OK. Will be changed.